src/components/expression/expression-renderer.js

```diff
@@ -318,7 +359,28 @@ genotet.ExpressionRenderer.prototype.drawMatrixCells_ = function() {
     transformLeft += this.DEFAULT_PROFILE_MARGIN;
   }
   if (this.data.options.showConditionLabels) {
-    transformTop += this.LABEL_MARGIN_;
+    transformTop += this.conditionLabelHeight_ +
+      this.LABEL_MARGIN_ + this.LABEL_DIFFERENCE_;
+  }
+
+  if (this.data.clickedCell) {
+    this.data.clickedCell.row = heatmapData.geneNames.indexOf(
+      this.data.clickedCell.geneName
+    );
+    this.data.clickedCell.column = heatmapData.conditionNames.indexOf(
+      this.data.clickedCell.conditionName
+    );
+    if (
+      this.data.clickedCell.row == -1 || this.data.clickedCell.column == -1
```

Comment:
The first condition should be on the first line, and the second on the second line. The OR operator is at the end of the first line.